src/type/scalars.js

```diff
@@ -121,3 +144,45 @@ export const GraphQLID = new GraphQLScalarType({
       null;
   }
 });
+
+export const GraphQLDateTime = new GraphQLScalarType({
+  name: 'DateTime',
+  description: 'An ISO-8601 encoded UTC date string.',
+  serialize(value: mixed): string {
+    if (!(value instanceof Date)) {
```

Comment:
Good point! We should promote the use of this scalar by making it very convenient to link to any dateTime representation that are used by the underlying systems: `Date`, ISO string or unix timestamp. I will include support for all of these!

Thanks for the great tips!